dashboard/app/models/lti.rb

```diff
@@ -0,0 +1,5 @@
+module Lti
+  def self.table_name_prefix
```

Comment:
Cool trick with the prefix. Wish we had modularized the other tables to start with!

Reply:
We could still apply modularization to the existing models